recipes/r-xslt/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set version = '1.4.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-xslt
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/xslt_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/xslt/xslt_{{ version }}.tar.gz
+  sha256: 64aacaff2dacb474c3abe2e91589676f8741981011ca7fc35b814e0ceb884c31
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```

Let's skip Windows. The package source will need to be patched to get it working and this is something that can be postponed until after feedstock creation.

```suggestion
  skip: True  # [win]
  number: 0
```